app/controllers/Application.scala

```diff
@@ -79,13 +80,21 @@ class Application(
       title,
       id,
       js,
-      contributionsPayPalEndpoint,
+      paymentApiPayPalEndpoint = paymentApiUrl.concat(paymentApiPayPalCreatePaymentPath),
```

Comment:
Is it possible to concat these before they are passed in to this class? Then there is one fewer parameter in the constructor and we don't need to concat twice in the class.